travis-ymls/logbook.trvis.yml

```diff
@@ -0,0 +1,84 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : logbook
+# Source Repo         : https://github.com/getlogbook/logbook.git
+# Travis Job Link     : https://travis-ci.com/github/santosh653/logbook
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+arch:
+- amd64
+- ppc64le
+language: python
+dist: xenial
+addons:
+  apt:
+    sources:
+      - chris-lea-redis-server
+      - sourceline: 'ppa:chris-lea/zeromq'
+    packages:
+    - redis-server
+    - libzmq3-dev
+services:
+- redis-server
+#Running latest python versions
```

Comment:
Does 2.7 fail for intel also ?

Reply:
@gururajrkatti No, Guru. On Intel it is running fine the python2.7,. But the Latest Pythons versions are fine.,

Reply:
@gururajrkatti Can you review this PR & approve? This is old one.

Reply:
@gururajrkatti , Since It is running fine on Latest versions of python3, Is it fine to proceed with PR? Could not figure out the failure on python7 & it is very old PR.

Reply:
We can not close with an unresolved ppc64le specific failure